src/operator/src/metrics.rs

```diff
@@ -52,8 +52,8 @@ lazy_static! {
     )
     .unwrap();
     pub static ref CREATE_OR_ALTER_TABLES: HistogramVec = register_histogram_vec!(
-        "greptime_create_or_alter_tables",
-        "grpc region request",
+        "greptime_table_operator_create_or_alter_tables",
```

Comment:
Maybe add the `on_demand` suffix
```suggestion
        "greptime_table_operator_create_alter_on_demand",
```